### PR TITLE
Add flags to the keras eval script.

### PR DESCRIPTION
**Pull request by LucasSloan**
Currently it hardcodes the location of the COCO dataset and the model to use.

### CODE REVIEW

File: efficientdet/keras/eval.py [hunk 1]
```diff
@@ -26,16 +27,36 @@
...
+    'COCO validation JSON containing golden bounding boxes.')
+flags.DEFINE_string('model_name', 'efficientdet-d0',
+                    'Model name: the efficientdet model to use.')
+flags.DEFINE_string('checkpoint', None, 'Location of the checkpoint to evaluate.')
+
+FLAGS = flags.FLAGS
+
 def main(_):
-  config = hparams_config.get_efficientdet_config('efficientdet-d0')
+  if FLAGS.validation_file_pattern is None:
```
Comment (mingxingtan) [inline efficientdet/keras/eval.py:42]: You can use flags.mark_flag_as_required('validation_file_pattern')
see: https://abseil.io/docs/python/guides/flags

**Comment by mingxingtan** [comment]
Thanks Lucas. Would be great if you can also help update the infer.py file.

**Comment by LucasSloan** [comment]
Would you like me to update infer.py in this PR?

**Comment by mingxingtan** [comment]
Yep, the change should be very similar, no?

File: efficientdet/keras/eval.py (hunk 1 above)
Comment (LucasSloan) [reply efficientdet/keras/eval.py]: Done.

File: efficientdet/keras/eval.py [hunk 2]
```diff
@@ -15,36 +15,53 @@
...
+flags.DEFINE_string('validation_file_pattern', None,
+                    'Glob for evaluation tfrecords (e.g., COCO val2017 set)')
+flags.DEFINE_string(
+    'val_json_file', None,
+    'COCO validation JSON containing golden bounding boxes.')
+flags.DEFINE_string('checkpoint', None, 'Location of the checkpoint to evaluate.')
+flags.DEFINE_string('model_name', 'efficientdet-d0',
+                    'Model name: the efficientdet model to use.')
+
+flags.mark_flag_as_required('validation_file_pattern')
```
Comment (mingxingtan) [inline efficientdet/keras/eval.py:40]: You need to move this right after "if __name__ == '__main__'"

File: efficientdet/keras/eval.py [hunk 3]
```diff
@@ -15,36 +15,53 @@
...
 from keras import anchors
 from keras import efficientdet_keras
 from keras import postprocess
 
+flags.DEFINE_string('validation_file_pattern', None,
+                    'Glob for evaluation tfrecords (e.g., COCO val2017 set)')
+flags.DEFINE_string(
+    'val_json_file', None,
+    'COCO validation JSON containing golden bounding boxes.')
+flags.DEFINE_string('checkpoint', None, 'Location of the checkpoint to evaluate.')
```
Comment (mingxingtan) [inline efficientdet/keras/eval.py:36]: You can use yapf to format your code (max length is 80 characters)

Comment (LucasSloan) [reply efficientdet/keras/eval.py:36]: I tried running yapf on it and it went to 4 space indentation - is that correct?

File: efficientdet/keras/eval.py (hunk 2 above)
Comment (LucasSloan) [reply efficientdet/keras/eval.py]: Done.

File: efficientdet/keras/eval.py (hunk 3 above)
Comment (mingxingtan) [reply efficientdet/keras/eval.py:36]: No, it shouldn't. You need to use style=yapf, like here: https://github.com/google/automl/blob/master/efficientdet/g3doc/faq.md#21-how-can-i-format-my-code-for-prs

Comment (LucasSloan) [reply efficientdet/keras/eval.py]: Got it!

**Pull request merged**